### PR TITLE
Bump kedro-viz version

### PR DESCRIPTION
**Pull request by AntonyMilneQB**
[kedro-viz 6.0.0 has been released](https://github.com/kedro-org/kedro-viz/pull/1297) but is not actually a breaking release from a kedro user perspective.

We should bump the requirement in our spaceflights starter so that users get the latest version.

See also https://github.com/kedro-org/kedro/pull/2469.

### CODE REVIEW

**Review by tynandebold — APPROVED** [review]
Looks good!

It's still not on PyPI though, FYI.

**Comment by AntonyMilneQB** [comment]
Ah ok, we'll need to wait until it's hit PyPI before CI here passes and we can merge.

**Comment by astrojuanlu** [comment]
There it is! https://pypi.org/project/kedro-viz/6.0.0/

**Review by stichbury — APPROVED** [review]

**Pull request merged**
